sstables/sstable_directory.cc

```diff
@@ -166,12 +180,16 @@ sstable_directory::process_sstable_dir(process_flags flags) {
 
     scan_state state;
 
-    directory_lister sstable_dir_lister(_sstable_dir, lister::dir_entry_types::of<directory_entry_type::regular>(), &manifest_json_filter);
+    components_lister sstable_dir_lister(_sstable_dir);
```

Comment:
I suppose later this will come from a higher layer and it will be a polymorphic type.

Reply:
Yes